src/main/java/org/gitlab/api/GitlabAPI.java

```diff
@@ -674,10 +674,11 @@ public GitlabProject getProject(String namespace, String projectName) throws IOE
     /**
      * Get a list of the namespaces of the authenticated user.
      * If the user is an administrator, a list of all namespaces in the GitLab instance is shown.
+     *
```

Comment:
It's generally better to edit these into the original commit(s) using git rebase.

Reply:
yes, I will squash it.